homeassistant/loader.py

```diff
@@ -826,8 +834,47 @@ async def resolve_dependencies(self) -> bool:
 
         return self._all_dependencies_resolved
 
+    async def async_get_component(self) -> ComponentProtocol:
+        """Return the component.
+
+        This method will load the component if it't not already loaded
```

Comment:
If it's not already

Reply:
https://github.com/home-assistant/core/pull/111560